Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 apache-superset = "==3.0.1"
 cython = "==3.0.3"
+holidays = "==0.23"
```

Comment:
Version has to be set explicitly (newer version is not compatible with prophet).